README.md

```diff
@@ -19,3 +19,8 @@ Read below guides to find out how we use nuxt;
 Enjoy your web site project 🤗
 
 To contribute please read [Contributing](CONTRIBUTING.md).
+
+```mermaid
+flowchart
+    A --> B
+```
```

Comment:
if you've reproduced the problem under the `test` folder, please remove it from here